### DISCUSSION TITLE
link again external libraries

### DISCUSSION THREAD

**Discussion by bdahlendr** [microsoft/vscode-cpptools#7769] [General]
Hello,
i wan't to add some external libs into a gcc and a msvc project, with the c/c++ extension but i can't find any document which show how can i add the libs.
Can anybody give me an example/document?

Bests 

Brian

**Comment by sean-mcmanus** [comment]
Linking to external libs is controlled by your compiler by passing various arguments to it such as `-I` or `-l`. Your build system such as makefile or CMake may also have settings for this. The C/C++ extension doesn't have any features for linking external libraries -- you normally just add the library's path to the includePath setting in c_cpp_properties.json (for IntelliSense) and/or the "args" in tasks.json (for building).
